src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -519,15 +519,15 @@ ncclResult_t mscclEnqueueCheck(
     case mscclNoGroup:
 #ifdef ENABLE_MSCCLPP
       if (comm->mscclppCompatible) {
-        if (threadLocalStatus.captureStatus == mscclUnknownCaptureStatus) {
-          INFO(NCCL_COLL, "MSCCL++: reading capture status");
-          NCCLCHECK(mscclGetCaptureStatus(comm->rank, stream));
-        }
+        //if (threadLocalStatus.captureStatus == mscclUnknownCaptureStatus) {
+        INFO(NCCL_COLL, "MSCCL++: reading capture status");
+        NCCLCHECK(mscclGetCaptureStatus(comm->rank, stream));
+        //}
```

Comment:
I'd also suggest that, if we read the capture status for every collective, no longer a need to log it here (or could convert to `TRACE`).
```suggestion
        INFO(NCCL_COLL, "MSCCL++: reading capture status");
        NCCLCHECK(mscclGetCaptureStatus(comm->rank, stream));
```